en/README.md

```diff
@@ -68,13 +68,9 @@ Clojure | [clj-mavlink](https://github.com/WickedShell/clj-mavlink) | Y | Y | Y
 
 ## Forums and Chat {#support}
 
-The core development team and community are active on the following chat channel:
+The core development team and community are active on the following mailing list:
```

Comment:
So the story is everything goes via the mailing list? I like that - It is neutral and takes away all the questions about specific forums.